final/bitcoind/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+cp -u /bitcoin.conf $BITCOIN_DATA/
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
+  echo "$0: assuming arguments for bitcoind"
+
+  set -- bitcoind "$@"
+fi
+
+if [ $(echo "$1" | cut -c1) = "-" ] || [ "$1" = "bitcoind" ]; then
```

Comment:
why is this block conditional?  seems like we always want to do it

Reply:
we won't need `datadir` for normal commands though

Reply:
what do you mean by "normal commands"?

Reply:
`ls` or `cat` when you call `docker container bitcoind exec ls /`

Reply:
I see.  well, in that case we can leave it as it is.